Components/Allemansrätt.js

```diff
@@ -0,0 +1,55 @@
+import React from 'react';
+import { View, FlatList, StyleSheet, Text } from 'react-native';
+import styled from 'styled-components/native';
+
+export const Allemansrätt = () => {
+  return (
+    <Container>
+      <Emojis> 🌲 🌲 🌲 </Emojis>
+      <InnerContainer>
+        <Title>About the Allemansrätt</Title>
+        <Title2>
+          Allemansrätten Sveriges natur är öppen för alla tack vare
+          allemansrätten.
+        </Title2>
+        <Text>
+          Det betyder att du utan bekymmer kan vistas i de flesta naturområden,
+          även i privatägd skog och mark. Ledorden “inte störa, inte förstöra”
+          summerar allemansrätten väl.
+        </Text>
+      </InnerContainer>
+    </Container>
+  );
+};
+
+const Title = styled.Text`
+  font-size: 30px;
+  text-align: center;
+  color: #1e5f18;
+`;
+
+const Title2 = styled.Text`
+  font-size: 15px;
+  text-align: left;
+  margin-bottom: 10px;
+
+  color: palevioletred;
```

Comment:
I realised that in almost all the styled components when added this particular color, there is a blank line on top of it.

Reply:
hmm that's weird, I don't know why I did that, but thank you for pointing it out =)